sale_order_line_dates/README.rst

```diff
@@ -0,0 +1,43 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+=====================
+Sale order line dates
+=====================
+
+
+This module adds requested date to a sales order lines and propagate it to
+stock moves and procurements
```

Comment:
Can you clarify how this relates to the sale order's requested_date field from sale_order_dates?